relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -689,8 +689,9 @@ fn extract_span_metrics(
     Ok(())
 }
 
-static SQL_TABLE_EXTRACTOR_REGEX: Lazy<Regex> =
-    Lazy::new(|| Regex::new(r#"(?i)(from|into)(\s|\()+(?P<table>(\w+(\.\w+)*))(\s|\))+"#).unwrap());
+static SQL_TABLE_EXTRACTOR_REGEX: Lazy<Regex> = Lazy::new(|| {
+    Regex::new(r#"(?i)(from|into)(\s|"|\()+(?P<table>(\w+(\.\w+)*))(\s|"|\))+"#).unwrap()
```

Comment:
Just a quick question, do we always have `""` around the table name , and no other types of quotes are used? Can it be there will be no quotes at all?